api/src/services/SolrIndexer.ts

```diff
@@ -322,12 +322,12 @@ class SolrIndexer {
         }
 
         // Change tracker details:
-        const change = await this.getChangeTrackerDetails(
-            pid,
-            fields["fgs.lastModifiedDate_txt_mv"][0] ?? "1900-01-01T00:00:00Z"
-        );
-        fields.first_indexed = change.getFirstIndexed;
-        fields.last_indexed = change.getLastIndexed;
+        // const change = await this.getChangeTrackerDetails(
```

Comment:
This change was made to work around the fact that you don't have a working VuFind instance in your test environment, but it should not be committed here; I suspect this was included by accident.

A better solution to this problem would be to adjust `getChangeTrackerDetails` so that if the VuFind URL is set to false in the configuration, it returns false or null, and then skipping these fields if VuFind is not configured. That would make it easier to test without a full VuFind, and also without requiring code-hacking to avoid errors. Maybe that's worth doing in a separate PR.